src/pb/GridFunc.cc

```diff
@@ -373,14 +372,15 @@ void GridFunc<T>::setValues(const T val)
 
     for (int i = 0; i < n; i++)
         pu[i] = val;
+
+    updated_boundaries_ = true;
```

Comment:
Was this a bug, since we were not doing this before?

Reply:
It was not a bug. But by not doing that, we were triggering ghost values filling later when not needed (since ghost values are set here)